### PR TITLE
Migrate CI to github actions

### PR DESCRIPTION
**Pull request by RobbeSneyders**
This PR moves the CI pipeline from the Zalando's Travis CI to github actions so it can run on this repository.

Since no upper limits were defined for the dependencies, the pipeline was broken and I had to implement some fixes first. I added upper limits for the dependencies to prevent external changes from breaking Connexion in the future.

### CODE REVIEW

**Comment by MatthiasDurivet** [comment]
> This PR moves the CI pipeline from the Zalando's Travis CI to bitbucket pipeline so it can run on this repository.

Did you mean to GitHub Actions?

**Comment by MatthiasDurivet** [comment]
So the big change is the pipeline was converted to GitHub Actions (what was it before that?)

Also some minor code/import changes to make sure the pipeline succeeds?

If so LGTM

File: tests/fakeapi/aiohttp_handlers.py [hunk 1]
```diff
@@ -78,6 +78,22 @@ async def aiohttp_token_info(token_info):
...
+
+
+async def aiohttp_bearer_auth(token_info):
+    return await aiohttp_token_info(token_info)
+
+
+async def aiohttp_async_bearer_auth(token_info):
+    return await aiohttp_token_info(token_info)
+
+
```
Comment (MatthiasDurivet) [inline tests/fakeapi/aiohttp_handlers.py:96]: What is this solving?
Should all of these be async or only the ones with async in the name?

**Review by MatthiasDurivet — APPROVED** [review]
LGTM

File: setup.py [hunk 2]
```diff
@@ -21,19 +21,19 @@ def read_version(package):
...
-    'jsonschema>=2.5.1',
-    'PyYAML>=5.1',
-    'requests>=2.9.1',
-    'inflection>=0.3.1',
-    'openapi-spec-validator>=0.2.4',
+    'clickclick>=1.2,<21',
+    'jsonschema>=2.5.1,<4',
+    'PyYAML>=5.1,<6',
+    'requests>=2.9.1,<3',
+    'inflection>=0.3.1,<1',
```
Comment (Ruwann) [inline setup.py]: Given that it's still on 0.x , breaking changes could still occur within the same major version, so perhaps we should be a bit more strict for this one and specify `<0.4` ?

**Review by Ruwann — APPROVED** [review]

File: tests/fakeapi/aiohttp_handlers.py (hunk 1 above)
Comment (RobbeSneyders) [reply tests/fakeapi/aiohttp_handlers.py:96]: The operationId in the OpenAPI spec needs to be unique to be valid. Connexion uses the operationId to link to the Python code that needs to be executed. Previously all endpoints in [tests/fixtures/aiohttp/openapi_secure.yaml](https://github.com/ml6team/connexion/pull/2/files#diff-6ccbde14498e093481f6333cbe9ba55610f1b70234bbd20311c0bece521a3653R11) pointed to the same Python function which led to validation errors of the spec, so I decoupled this into separate functions.

File: setup.py (hunk 2 above)
Comment (RobbeSneyders) [reply setup.py]: I've updated the upper limits for 0.x versions to the minor version above the current latest version. In this case, inflection is already on 0.5.1, so I used 0.6.

**Comment by svetlana-v** [comment]
Sorry for the late comment. I think you forgot to remove `Travis CI build status` related info from README.rst?

**Pull request merged**